IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift

```diff
@@ -192,4 +192,11 @@ public class ReachFive: NSObject {
                 }
         }
     }
+    
+    public func listWebAuthnDevices(authToken: AuthToken) -> Future<[DeviceCredential], ReachFiveError> {
+         
+           return self.reachFiveApi
+            .getWebAuthnRegistrations(authorization: authToken.tokenType! + " " + authToken.accessToken)
+                .map { $0 }
```

Comment:
Swift Q: what does this map do?

Reply:
`$0` is a short-hand for the first argument of the anonymous function (I had to look that up [here](https://docs.swift.org/swift-book/LanguageGuide/Closures.html#ID100)) so it seems it does nothing.

Reply:
I'd prefer that we don't construct the Authorization as a `String` here to pass it down the stack.